app/assets/stylesheets/form.scss

```diff
@@ -4,49 +4,69 @@
 }
 
 @mixin form-inputs {
-  width: 250px;
   border-radius: 5px;
+  width: 250px;
 }
 
 .form-inputs {
-  @include left-margin-padding;
-  border-radius: 10px;
+  border-radius: 5px;
```

Comment:
Properties should be ordered background-color, border-radius, margin-left, padding